components/Header.tsx

```diff
@@ -12,6 +28,8 @@ function Header({
   widerContent?: boolean;
 }): React.ReactElement {
   const [menuOpen, setMenuOpen] = useState(false);
+  const [showDeploy, setShowDeploy] = useState(false);
+  useEffect(() => setShowDeploy(displayDeploy()), []);
```

Comment:
Seems like it's simpler to use `useMemo`

```suggestion
  const showDeploy = useMemo(() => displayDeploy(), []);
```

Reply:
`useMemo()` will also run on the server.

Reply:
Ah, ok. makes sense!